.circleci/test-deploy.yml

```diff
@@ -83,13 +83,29 @@ workflows:
           cypress-cache-key: cypress-cache{{ arch }}-{{ checksum "examples/yarn-install/package.json" }}
           post-install: "npx cypress install"
           package-manager: "yarn"
+      - cypress/run:
+          filters: *filters
+          name: Yarn Version Example
+          working-directory: examples/yarn-install
+          cypress-cache-key: cypress-cache{{ arch }}-{{ checksum "examples/yarn-install/package.json" }}
+          post-install: "./check-yarn-version.sh 1.22.5 && npx cypress install"
```

Comment:
If you intend to support Yarn Modern Plug'n'Play then you can't use `npx`. It has to be `yarn`.